ctapipe/io/containers.py

```diff
@@ -542,120 +542,44 @@ class SST1MDataContainer(DataContainer):
+    ring_radius = Field(nan * u.deg, "radius of the fitted muon ring", unit=u.deg)
+    ring_center_phi = Field(
+        nan * u.deg, "Angle of ring center within camera plane", unit=u.deg
+    )
+    ring_center_distance = Field(
+        nan * u.deg, "Distance of ring center from camera center", unit=u.deg
+    )
+    ring_chi2_fit = Field(nan, "chisquare of the muon ring fit", unit=u.deg)
+    ring_cov_matrix = Field(
+        np.full((3, 3), nan), "covariance matrix of the muon ring fit"
+    )
+    ring_containment = Field(nan, "containment of the ring inside the camera")
 
 
 class MuonIntensityParameter(Container):
-    """
-    Storage of muon intensity fit output
-
-    Parameters
-    ----------
-
-    obs_id : int
-        run number
-    event_id : int
-        event number
-    tel_id : int
-        telescope ID
-    impact_parameter: float
-        reconstructed impact parameter
-    impact_parameter_chi2:
-        chi squared impact parameter
-    intensity_cov_matrix:
-        Covariance matrix of impact parameters or alternatively:
-        full 5x5 covariance matrix for the complete fit (ring + impact)
-    impact_parameter_pos_x, impact_parameter_pos_y:
-        position on the mirror of the muon impact
-    COG_x, COG_y:
-        center of gravity
-    optical_efficiency_muon:
-        optical muon efficiency from intensity fit
-    ring_completeness:
-        completeness of the ring
-    ring_pix_completeness:
-        pixel completeness of the ring
-    ring_num_pixel: int
-        Number of pixels composing the ring
-    ring_size:
-        ring size
-    off_ring_size:
-        size outside of the ring
-    ring_width:
-        ring width
-    ring_time_width:
-        standard deviation of the photons time arrival
-    prediction: dict
-        ndarray of the predicted charge in all pixels
-    mask:
-        ndarray of the mask used on the image for fitting
-
-    """
-
-    obs_id = DeprecatedField(
-        0, "run identification number", reason="moved to event.index"
-    )
-    event_id = DeprecatedField(
-        0, "event identification number", reason="moved to event.index"
-    )
-    tel_id = DeprecatedField(
-        0, "telescope identification number", reason="moved to event.index"
-    )
-    ring_completeness = Field(0.0, "fraction of ring present")
-    ring_pix_completeness = Field(0.0, "fraction of pixels present in the ring")
-    ring_num_pixel = Field(0, "number of pixels in the ring image")
-    ring_size = Field(0.0, "size of the ring in pe")
-    off_ring_size = Field(0.0, "image size outside of ring in pe")
-    ring_width = Field(0.0, "width of the muon ring in degrees")
-    ring_time_width = Field(0.0, "duration of the ring image sequence")
+    ring_completeness = Field(nan, "fraction of ring present")
+    ring_pix_completeness = Field(nan, "fraction of pixels present in the ring")
+    ring_num_pixel = Field(-1, "number of pixels in the ring image")
```

Comment:
why not `nan` as well?

Reply:
there is no nan for integers

Reply:
nan implies float or double, this is a positive integer so -1 as missing value marker makes more sense